test/zdtm.py

```diff
@@ -481,6 +484,13 @@ def start(self):
             # move into some semi-random state
             time.sleep(random.random())
 
+        if self.__flavor.ns:
```

Comment:
Does it work for uns?

Reply:
yes

Reply:
fedora-asan tests fail for some reason
https://api.travis-ci.org/v3/job/614299079/log.txt